st2client/st2client/commands/action.py

```diff
@@ -246,22 +246,6 @@ class ActionRunCommandMixin(object):
     def get_resource(self, ref_or_id, **kwargs):
         return self.get_resource_by_ref_or_id(ref_or_id=ref_or_id, **kwargs)
 
-    @add_auth_token_to_kwargs_from_cli
```

Comment:
Also, minor style / consistency thing - we tend to put "public" methods at the top and "private" at the bottom / after "public" ones.